test/query/templates/GraqlLangTest.java

```diff
@@ -27,64 +27,74 @@
     private void runQuery(GraknClient.Transaction transaction, GraqlQuery query) {
```

Comment:
We're passing down transaction from the outside but we seem to be closing them inside. This feels off. Let me chat to you in person.

Reply:
Has been fixed in https://github.com/graknlabs/docs/pull/489/commits/31abfad06ae358a6b58c706d788cd07e9cf9e8b0